src/reducers/existingThoughtMove.ts

```diff
@@ -32,6 +32,7 @@ type RecursiveMoveResult = {
   archived?: Timestamp,
   contextsNew: Context[],
   contextsOld: Context[],
+  hashedKey: string,
```

Comment:
Let's call this `thoughtEncoded` to be consistent with similar variables used in other part of the codebase.